platform/darwin/src/MGLStyle.h

```diff
@@ -551,7 +551,7 @@ MGL_EXPORT
  
  This method automatically modifies the text property of any symbol style layer
  in the style whose source is the
- <a href="https://www.mapbox.com/vector-tiles/mapbox-streets-v7/#overview">Mapbox Streets source</a>.
+ <a href="https://www.mapbox.com/vector-tiles/mapbox-streets-v8/#overview">Mapbox Streets source</a>.
```

Comment:
Perhaps we should link instead to [an unversioned URL](https://www.mapbox.com/vector-tiles/mapbox-streets/):

https://github.com/mapbox/mapbox-gl-native/blob/033d16ba8614d7d5541cf69a05074398e35ad699/platform/darwin/src/MGLVectorTileSource.h#L39

However, that URL currently redirects to the v7 documentation, since v8 is still officially in beta.

Reply:
>However, that URL currently redirects to the v7 documentation, since v8 is still officially in beta.

Not for long @1ec5! 🎉 https://github.com/mapbox/vector-tiles/pull/67